### ISSUE TITLE
[Check] Reflected input into Django Responses

### ISSUE THREAD

**Issue by minusworld** [returntocorp/sgrep-rules#215]
**Description**
Based off this finding here:
```python
### https://github.com/edx/edx-platform/pull/19519/files
    user_filter = urllib.unquote(urllib.quote_plus(request.GET.get("user", "")))
    if not user_filter:	    if not user_filter:
        msg = _("user is not given.")	        msg = _("user is not given.")
        return HttpResponseBadRequest(msg)	        return HttpResponseBadRequest(msg)
    try:	    try:
        user = User.objects.get(Q(email=user_filter) | Q(username=user_filter))	        user = User.objects.get(Q(email=user_filter) | Q(username=user_filter))
    except User.DoesNotExist:	    except User.DoesNotExist:
        return HttpResponseBadRequest(_("user '{user}' does not exist").format(user=user_filter))	        return HttpResponseBadRequest(_("user '{user}' does not exist").format(user=user_filter))
```

- [x] This ticket has links, references, or examples.
- [x] Your check has true positive and true negative test cases.
- [ ] Your check has been labeled with true positive and false positive findings on at least 10 repositories, and the link is pasted in the ticket.
- [ ] Your check PR has been reviewed and merged.
Now, close the ticket!

**Comment by minusworld** [comment]
https://dev.massive.ret2.co/triager/filter/818 -> HttpResponse

**Comment by minusworld** [comment]
https://dev.massive.ret2.co/triager/filter/821 --> HttpResponseBadRequest
